elephant/asset.py

```diff
@@ -1233,56 +1201,59 @@ def _pmat_neighbors(mat, filter_shape, nr_largest):
         for x in bin_range_x:
             patch = mat[y: y + l, x: x + l]
             mskd = np.multiply(filt, patch)
-            largest_vals = np.sort(mskd, axis=None)[-nr_largest:]
+            largest_vals = np.sort(mskd, axis=None)[-n_largest:]
             lmat[:, y + (l // 2), x + (l // 2)] = largest_vals
 
     return lmat
 
 
 def joint_probability_matrix(
-        pmat, filter_shape, nr_largest=None, alpha=0, pvmin=1e-5,
+        pmat, filter_shape, n_largest=None, alpha=0, p_value_min=1e-5,
```

Comment:
Parameter name should be `min_p_value` according to the naming conventions.

Reply:
right